src/test/scala/ai/newmap/interpreter/TestNewmapFiles.scala

```diff
@@ -0,0 +1,61 @@
+package ai.newmap.interpreter
+
+import ai.newmap.interpreter.Lexer.{DQuote, Enc, Identifier, Number, Symbol}
+import org.scalatest._
+import ai.newmap.model._
+import ai.newmap.util.{Failure, Success}
+import scala.io.Source
+
+
+class TestNewmapFiles extends FlatSpec{
```

Comment:
Are we only testing the lexer in this file? Maybe name it accordingly

Reply:
I was thinking of adding the parsing tests in the same file.